test/fail_compilation/fail20730.d

```diff
@@ -0,0 +1,16 @@
+/*
+EXTRA_SOURCES: imports/test20730.d
+REQUIRED_ARGS: -o-
+PERMUTE_ARGS:
+TEST_OUTPUT:
+---
+fail_compilation/fail20730.d(15): Error: undefined identifier `undef20730`
+---
+*/
+import imports.test20730;
+
+void test20730()
```

Comment:
You can simplify the test by using the import module, but placing this function before struct File.